mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/16 - buy mode

Closes #16 

To test:
- Run any scenario with hardware to buy (like introductory scenario)
- Go to Buy view and click on button to buy hardware
- Move mouse around in office view and make sure a mesh of the hardware follows the cursor and a quad highlights the workspace the hardware will be placed in
- Repeat and mouse over the workplace the previous hardware was placed in and make sure the workspace highlighter turns red
